test/unit_tests/modules/beta-banner-state-spec.js

```diff
@@ -7,14 +7,16 @@ var sinon = require( 'sinon' );
 var StorageMock = require( '../fixtures/StorageMock' );
 
 var BASE_JS_PATH = '../../../cfgov/unprocessed/js/';
-var betaBanner;
+var globalBanner;
+var clickEvent;
 var contentDom;
 var contentAnimatedDom;
+var GlobalBanner;
 var targetDom;
-var clickEvent;
+var toggleStoredStateSpy;
 var webStorageProxy;
 
-describe( 'Beta Banner State', function() {
+describe( 'Global Banner State', function() {
```

Comment:
Oops, sure should! Fixed in https://github.com/cfpb/cfgov-refresh/pull/1688/commits/0b1e71b6209e2efef2405eda58ee1f960bf709d5